pattern-library/js/edx-icons.js

```diff
@@ -0,0 +1,15 @@
+define([], function() {
+
+    AFontGarde('edx-icons', {
+        // Add all of the individual icons you want to check for.
+        // I don't think we need to list them all.
+        glyphs: '\uE621\uE622\uE623',
```

Comment:
This is needed to actually run A Font Garde. We toss in a few random font icons. It uses these to see if the font is available.